### DISCUSSION TITLE
Rotation of div and background-image with svg

### DISCUSSION THREAD

**Discussion by ballotade** [dompdf/dompdf#2977] [Q&A] [answered]
Hello :)

I want to rotate a div:

`<div style="transform: rotate(10deg); ">Text</div>`

But it is not working.
Is it possible to rotate in dompdf and  if yes, what am I doing wrong?

Then I want to set an background-image to a div with base64_encode.
It ist working with png and jpg but with svg it is not working. 

Is there something I miss? 

Thank you for your help. I'm very grateful. :)

**Comment by bsweeney** [comment, marked as answer]
Rotation works OK for me. What version of Dompdf are you using?

As far as the SVG issue ... it looks like we never added support for using SVG as a background. At least, not in my limited testing and code review.

**Reply by bsweeney** [reply]
I went ahead and created #2978 to remind us to add support for SVG background images.

**Reply by ballotade** [reply]
Rotation is working. I made a mistake. Thank you.

And for the svg it would be great if it where supported. :)
